integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/core/EDUCoreSender.java

```diff
@@ -23,52 +27,58 @@
 
     private static final Logger log = LoggerFactory.getLogger(EDUCoreSender.class);
 
-    private IntegrasjonspunktProperties properties;
-    private ServiceRegistryLookup serviceRegistryLookup;
-    private StrategyFactory strategyFactory;
-    private Adresseregister adresseRegister;
-    private NoarkClient mshClient;
+    private final IntegrasjonspunktProperties properties;
+    private final ServiceRegistryLookup serviceRegistryLookup;
+    private final StrategyFactory strategyFactory;
+    private final Adresseregister adresseRegister;
+    private final NoarkClient mshClient;
+    private final InternalQueue internalQueue;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "internalQueue" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1068)